script/mhd

```diff
@@ -22,6 +22,12 @@ if [[ $gulp_args =~ ^open:.* ]]; then
   vault_opts="--duration=1h"
 fi
 # shellcheck disable=SC2086
-TS_NODE_TRANSPILE_ONLY=true MHD_ARGS="$mhd_args" \
-  aws-vault exec $vault_opts mindhive-ops -- \
+TS_NODE_TRANSPILE_ONLY=true
```

Comment:
I don't think this will get through as an env var to later subprocess calls. You need to `export` to do that.